src/main/java/kitchenpos/order/applicaiton/OrderService.java

```diff
@@ -0,0 +1,55 @@
+package kitchenpos.order.applicaiton;
+
+import kitchenpos.menu.domain.Menu;
+import kitchenpos.menu.domain.MenuRepository;
+import kitchenpos.order.domain.*;
+import kitchenpos.order.dto.OrderRequest;
+import kitchenpos.order.dto.OrderResponse;
+import kitchenpos.order.dto.OrderStatusRequest;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class OrderService {
+    private final MenuRepository menuRepository;
+    private final OrderRepository orderRepository;
+    private final OrderTableService orderTableService;
+
+    public OrderService(
+            final MenuRepository menuRepository,
+            final OrderRepository orderRepository,
+            final OrderTableService orderTableService) {
+        this.menuRepository = menuRepository;
+        this.orderRepository = orderRepository;
+        this.orderTableService = orderTableService;
+    }
+
+    @Transactional
+    public OrderResponse create(final OrderRequest orderRequest) {
+        OrderTable orderTable = orderTableService.findOrderTable(orderRequest.getOrderTableId());
+        List<Menu> menus = menuRepository.findAllById(orderRequest.findAllMenuIds());
+        Order order = orderRequest.toOrder(orderTable, OrderStatus.COOKING, menus);
+        order.checkValidOrder(menus.size());
```

Comment:
```suggestion
        Order order = orderRequest.toOrder(orderTable, OrderStatus.COOKING, menus);
```
주문 생성 시점에 유효성 검사를 해줄수 있을것같아요 😀

Reply:
반영하였습니다! :)